health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/presentation/ExerciseSampleApp.kt

```diff
@@ -57,8 +56,10 @@ fun ExerciseSampleApp(
 
         AppScaffold(
             timeText = {
-                if (ambientStateUpdate.ambientState is AmbientState.Interactive) {
-                    ResponsiveTimeText()
+                if (ambientStateUpdate.ambientState is AmbientState.Interactive &&
+                    currentScreen?.destination?.route != PreparingExercise.route
```

Comment:
You can hide this on that screen with ScreenScaffold instead.

Reply:
Adjusted